thermo_nerf/thermal_nerf/thermal_nerf_model.py

```diff
@@ -348,7 +348,8 @@ def get_image_metrics_and_images(
         )
         combined_thermal = torch.cat([gt_thermal, thermal], dim=1)
 
-        images_dict.update({"thermal": combined_thermal})
+        images_dict.update({"thermal": thermal})
```

Comment:
use your enum `RenderedImageModality` instead of a string here and elsewhere it's used? :)?